docs/user/dashboard/lens.asciidoc

```diff
@@ -402,3 +402,19 @@ Pagination in a data table is unsupported. To use pagination in data tables, cre
 ====
 Specifying the color for a single data point, such as a single bar or line, is unsupported.
 ====
+
+[discrete]
+[[is-it-possible-to-inspect-the-elasticsearch-queries-in-Lens]]
+.*Is it possible to inspect the Elasticsearch queries in Lens?*
+[%collapsible]
+====
+Using the Inspector within the editor or in the dashboard it is possible to select the specific request sent by the visualization to Elasticsearch.
```

Comment:
Maybe it's non-native speaker thing, but I find this sentence difficult to read with the current structure. How about: 
`You can inspect the requests sent by the visualization to Elasticsearch by using the Inspector. It can be accessed within the editor or in the dashboard.`

cc @KOTungseth